code/modules/mob/living/silicon/robot/dogborg/dog_sleeper_vr.dm

```diff
@@ -223,13 +231,32 @@
 		sleeperUI(usr)
 		return
 
+	if(href_list["sync"])
+		synced = TRUE
+		var/success = 0
+		for(var/obj/machinery/r_n_d/server/S in machines)
```

Comment:
It's the same thing in the regular portable DA, and there's no need for cooldown because it only shows up once if there are any new unsynced updates and immediately refreshes away on the click. It requires analyzing a new item in order to bring the button back